### PR TITLE
Parse and transform ACRO JSON into something more useful.

### LINKED ISSUE #99: Parse ACRO json into something easier to consume in UI
The ACRO json format is hard to consume, as it requires various processing before we have the data in a useful format. e.g

- reparse `outcome` from string into JSON
- parse the status out from "summary" text
- parse output type from "command" text
- split comments
- split cell redaction reasons

We have feedback for the ACRO team about making these changes to the JSON upstream, but we have not yet supplied that and don't know if it will happen or not.

The UI code is currently parsing some of this itself in Javascript.

We should instead transform the current ACRO json in python to the desired structure, and pass that to the UI as the JSON.

Doing it in python rather than makes it easier to develop, test, and reuse.

It also provides a useful abstraction barrier to protect UI code from upstream changes to the ACRO json as they happen - we can handle that change in python.

It might also provide a useful test of our proposed structure for upstream.

It also gives us a place to add in annotations from other sources (e.g. OpenSAFELY user info, or similar).

### PR DESCRIPTION
**Pull request by bloodearnest**
This make the JSON easier to consume in the UI code, and also provides a
single tested point to parse data or handle changes in format.

The only b/w incompat was renaming the filed containing the file path
from `output` to `path`, and the name output seems a little overused
already.

Fixes #99

### CODE REVIEW

File: sacro/transform.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            "name": name,
+            "display_name": display_name,
+            "type": output_type,
+            "status": status,
+            "summary": summary,
+            "outcome": json.loads(outcome) if isinstance(outcome, str) else outcome,
+            "path": metadata.get("output"),
+            "command": metadata.get("command"),
+            "comments": [
+                c.strip() for c in metadata.get("comments", "").split(",") if c.strip()
```
Comment (lucyb) [inline sacro/transform.py:43]: This line needs a test, as all the existing tests expect an empty list here.

**Review by lucyb — APPROVED** [review]
